spec/Cabin/Infotainment.vspec

```diff
@@ -266,4 +266,27 @@ SmartphoneProjection.SupportedMode:
   allowed: [ 'ANDROID_AUTO', 'APPLE_CARPLAY', 'MIRROR_LINK', 'OTHER' ]
   description: Supportable list for projection.
 
+SmartphoneScreenMirroring:
+  type: branch
+  description: All smartphone mirroring actions.
+
+SmartphoneScreenMirroring.Active:
+  datatype: string
+  type: actuator
+  allowed: [ 'NONE', 'ACTIVE',  'INACTIVE' ]
+  description: Mirroring activation info.
+  comment: NONE indicates that mirroring is not supported.
+
+SmartphoneScreenMirroring.Source:
+  datatype: string
+  type: actuator
+  allowed: ['USB', 'BLUETOOTH', 'WIFI']
+  description: Connectivity source selected for mirroring.
+
+SmartphoneScreenMirroring.Resolution:
+  datatype: string
+  type: attribute
+  allowed: [ '720p', '1080p', '4K' ]
```

Comment:
We have a recommendation at https://covesa.github.io/vehicle_signal_specification/rule_set/data_entry/allowed/#recommendation-on-string-values to use "enum-like" identifiers, i.e. something like `[A-Z][A-Z0-9_]*`.

If we want to follow that recommendation (which is a recommendation so we can ignore it if we have good reasons) we could consider having values like `['RES_720P', 'RES_1080P', 'RES_4K']` or something similar